### PR TITLE
Fixes on cache in reason of multiple subtitles

### PR DESCRIPTION
**Pull request by rpaggi**
I made a change in the cache, because i saw having several occurrences of the same film with various languages in subtitles, for example: 
{"Movie 1", "English: ..."} 
{"Movie 1", "English: ... Spanish: ..."} 
.
{"Movie 1", "English: ... Spanish ..., N '}

Now the cache verify if have an occurrence before insert in DB, if exists it makes a update, if not exists it makes a insert.

Enjoy

### CODE REVIEW

File: js/frontend/providers/cache.js [hunk 1]
```diff
@@ -63,7 +63,21 @@ App.Cache = {
...
+            //Implementation to check if exist registration in db and update instead of insert
+            //Does a SELECT on Provider
+            tx.executeSql('SELECT data FROM ' + provider + ' WHERE key = ?', [key], function (tx, results) {
+                try {
+                    if (results.rows.length) {
+                        tx.executeSql('UPDATE ' + provider + ' SET data = ? WHERE key = ?', [data, key]);
+                    } else {
+                        tx.executeSql('INSERT INTO ' + provider + ' VALUES (?, ?)', [key, data]);
+                    }
+                } catch (e) {
```
Comment (jeromelefeuvre) [inline js/frontend/providers/cache.js]: Why do you need a catch ? Possible to broke all ?

Comment (rpaggi) [inline js/frontend/providers/cache.js]: True, when i go to home i'll change this
Em 06/03/2014 12:25, "Jerome Lefeuvre" notifications@github.com escreveu:

Comment (rpaggi) [inline js/frontend/providers/cache.js]: Or if you can change for me :)
Em 06/03/2014 12:25, "Jerome Lefeuvre" notifications@github.com escreveu:

Comment (Patineta) [inline js/frontend/providers/cache.js]: Great! we'll wait until the next fix before merging.

**Comment by rpaggi** [comment]
Now i remove the catch, leaving only the try.

**Comment by Patineta** [comment]
Thanks for the bugfix!

**Pull request merged**